### PR TITLE
Packages for Geo-Django

### PR DESCRIPTION
**Pull request by athai**
Add packages required for GeoDjango (in order to use postgis).

### CODE REVIEW

**Comment by Kennric** [comment]
@ramereth can we get a quick look at this? This will let us get streamwebs-staging on app3 for Renee to review

**Comment by jldugger** [comment]
@athai so these packages are required for building certain python libs, correct?

**Comment by athai** [comment]
@jldugger Yeah. Most of them are needed for GeoDjango and PostGis, I think.

File: recipes/default.rb [hunk 1]
```diff
@@ -35,6 +35,12 @@
   package p
 end
 
+# geo-django depends
+%w(gdal-python geos-python proj proj-nad postgresql-devel freetype-devel
+   libjpeg-devel libpng-devel).each do |p|
```
Comment (jldugger) [inline recipes/default.rb]: On app3:

```
# yum info libjpeg-devel
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
Error: No matching Packages to list
```

**Comment by jldugger** [comment]
@athai so if libjpeg-devel wasn't required, where is this list coming from?

**Comment by athai** [comment]
@jldugger The list is coming from the StreamWebs dockerfile. I wasn't the one who added that package to the file, but I think someone needed it to work with static images?

**Comment by jldugger** [comment]
In a test VM, `yum install libjpeg-devel` prompted to install libjpeg-turbo-devel. Unsure why `yum info` returned nothing, when I would have expected it to mention it was a virtual package etc.

**Comment by jldugger** [comment]
@athai :+1: changes look fine. However, do not merge this. Follow https://docs.osuosl.org/config-management/chef/cookbooks/updating-organization-cookbooks.html instead

**Comment by athai** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.9.4.  Have a nice day!

**Pull request merged**